scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1238,6 +1313,13 @@ void layered_dispatch_no_topo(s32 cpu, struct task_struct *prev)
 		return;
 	}
 
+	dsq_id = cpu_hi_fallback_dsq_id(cpu);
```

Comment:
Why consume hi fallback before preempting layers? Or rather, shouldn't fallback layers also have a weight and mechanism to avoid starvation?

Reply:
I think one issue is that the vtime still gets charged to the layer when a task is put onto the fallback dsq. I guess those vtimes need to be tracked as well.